client/src/app/pages/repositories/Mvn.tsx

```diff
@@ -49,20 +49,18 @@ export const RepositoriesMvn: React.FC = () => {
   //     mvnForcedSettingMutation.mutate(!mvnForcedSetting.data);
   // };
 
-  const [storageValue, setStorageValue] = useState<string>();
+  let storageValue: string = "";
+  const cache = useFetchCache();
```

Comment:
What are your thoughts on destructuring the isSuccess & refetch values/methods here? 

Looking good!

Reply:
It's about "Type Narrowing", by keeping the context it help TS.
And it makes consuming requests easier since we don't have to rename anything.